CONTRIBUTING.md

```diff
@@ -209,6 +209,52 @@ this can be suppressed by adding this near the top of your file:
 #pragma warning disable SA1401 // Fields must be private
 ```
 
+### SafeHandles
+
+Safe handles should follow a few rules :
+* They should have an empty constructor that does nothing. (The marshaller will use this one when it need to create a SafeHandle)
+* They should have a constructor allowing to reuse pre-existing handles.
+* They should have a static field for each invalid values for easy access.
+
+Example :
```

Comment:
While I don't object to a doc or docs folder, I don't think this is the best use for it.
Since nearly all our actual SafeHandle-derived classes are perfectly reasonable samples we could refer to, I meant that we can remove the sample code altogether from this file and simply link to one of our actual SafeHandle classes in code. The way I do in this document already in the "Struct field types" section.

That way, we don't have to maintain the documentation every time we change something in the code to update, and folks are confident what they're looking at is current.